src/popup/components/LatestTransactionsCard.vue

```diff
@@ -30,7 +30,6 @@ import {
   ref,
   watch,
```

Comment:
```
Duplicate keys detected: 'ak_2AVeRypSdS4ZosdKWW1C4avWU4eeC2Yq7oP7guBGy8jkxdYVUy-th_2bVpja3SK2YAffwFrteGPpXxA9epcbsu5VxoXKbf9EPKqwaT5R-0'. This may cause an update error.
```

In case one subaccount sending to another latest transaction.
https://github.com/aeternity/superhero-wallet/blob/c0b57b68835d070d8af194bcbe3049459b458378/src/popup/components/LatestTransactionsCard.vue#L19

Reply:
i was unable to reproduce it log

Reply:
before that pr, problem can be exist, because "direction" not worked properly and code has wrong transaction Owner